packages/core/src/Account.ts

```diff
@@ -83,40 +82,6 @@ export enum ConnectionState {
 }
 
 export interface Account {
-  on(
-    event: PayloadBundleType.ASSET,
-    listener: (payload: OtrMessage.FileAssetMessage | OtrMessage.ImageAssetMessage) => void,
-  ): this;
-  on(event: PayloadBundleType.BUTTON_ACTION, listener: (payload: OtrMessage.ButtonActionMessage) => void): this;
-  on(event: PayloadBundleType.ASSET_ABORT, listener: (payload: OtrMessage.FileAssetAbortMessage) => void): this;
-  on(event: PayloadBundleType.ASSET_IMAGE, listener: (payload: OtrMessage.ImageAssetMessage) => void): this;
-  on(event: PayloadBundleType.ASSET_META, listener: (payload: OtrMessage.FileAssetMetaDataMessage) => void): this;
-  on(event: PayloadBundleType.CALL, listener: (payload: OtrMessage.CallMessage) => void): this;
-  on(event: PayloadBundleType.CLIENT_ACTION, listener: (payload: OtrMessage.ResetSessionMessage) => void): this;
-  on(event: PayloadBundleType.CLIENT_ADD, listener: (payload: UserMessage.UserClientAddMessage) => void): this;
-  on(event: PayloadBundleType.CLIENT_REMOVE, listener: (payload: UserMessage.UserClientRemoveMessage) => void): this;
-  on(event: PayloadBundleType.CONFIRMATION, listener: (payload: OtrMessage.ConfirmationMessage) => void): this;
-  on(event: PayloadBundleType.CONNECTION_REQUEST, listener: (payload: UserMessage.UserConnectionMessage) => void): this;
-  on(event: PayloadBundleType.USER_UPDATE, listener: (payload: UserMessage.UserUpdateMessage) => void): this;
-  on(
-    event: PayloadBundleType.CONVERSATION_CLEAR,
-    listener: (payload: OtrMessage.ClearConversationMessage) => void,
-  ): this;
-  on(event: PayloadBundleType.CONVERSATION_RENAME, listener: (payload: Events.ConversationRenameEvent) => void): this;
-  on(event: PayloadBundleType.LOCATION, listener: (payload: OtrMessage.LocationMessage) => void): this;
-  on(event: PayloadBundleType.MEMBER_JOIN, listener: (payload: Events.TeamMemberJoinEvent) => void): this;
-  on(event: PayloadBundleType.MESSAGE_DELETE, listener: (payload: OtrMessage.DeleteMessage) => void): this;
-  on(event: PayloadBundleType.MESSAGE_EDIT, listener: (payload: OtrMessage.EditedTextMessage) => void): this;
-  on(event: PayloadBundleType.MESSAGE_HIDE, listener: (payload: OtrMessage.HideMessage) => void): this;
-  on(event: PayloadBundleType.PING, listener: (payload: OtrMessage.PingMessage) => void): this;
-  on(event: PayloadBundleType.REACTION, listener: (payload: OtrMessage.ReactionMessage) => void): this;
-  on(event: PayloadBundleType.TEXT, listener: (payload: OtrMessage.TextMessage) => void): this;
-  on(
-    event: PayloadBundleType.TIMER_UPDATE,
-    listener: (payload: Events.ConversationMessageTimerUpdateEvent) => void,
-  ): this;
-  on(event: PayloadBundleType.TYPING, listener: (payload: Events.ConversationTypingEvent) => void): this;
-  on(event: PayloadBundleType.UNKNOWN, listener: (payload: any) => void): this;
```

Comment:
This is legacy behavior. We now get all events from the `onEvent` callback of the `listen` method